crates/store/re_chunk_store/src/writes.rs

```diff
@@ -62,7 +62,9 @@ impl ChunkStore {
         // * Somebody loads an old .rbl from somewhere and starts logging new blueprint data to it.
         // * Etc.
         if self.id.kind == re_log_types::StoreKind::Blueprint {
-            chunk = Arc::new(chunk.clone_as_untagged());
+            let patched = chunk.patched_for_blueprint_021_compat();
+            let patched = patched.clone_as_untagged();
+            chunk = Arc::new(patched);
         }
```

Comment:
I wish we had a version tag on the store so we're not applying this indefinitely into the future ...

Reply:
Yeah we need to pull the StoreInfos all the way in at some point. We can do that later though, that's fine (I don't expect this particular change to be particularly version specific -- `SpaceView` is not making a come back anytime soon I hope :stuck_out_tongue:).